YACS-RCOS/yacs.n
# Now Adding 404 Page!

**Issue**

Usually, when a user goes into the wilderness of the website, there was no page to alert them that they were lost, alone, and stuck forever. Now they can recover, come back to the main homepage and be free!

> closes #346 

**Test Procedure**

> 1. Go to website
> 2. Add `/asdfasdf` at the end of the URL (Or whatever you want to add at the end, shouldn't matter)
> 3. The 404-page should be there!

**Photos**
In light mode:
![image](https://user-images.githubusercontent.com/22281588/112244744-0ece9e80-8c26-11eb-996a-1e47a6013810.png)

In dark mode:
![image](https://user-images.githubusercontent.com/22281588/112244770-18580680-8c26-11eb-95c3-81be69a98358.png)


**Additional Info**
Would like any help to make it look more appealing! Thanks!
